### ISSUE TITLE
metrics show: error when type is specified but there is no xpath

### ISSUE THREAD

**Issue by efiop** [iterative/dvc#1506]
```
dvc run -M tmp.metric 'echo \{\"AUC\": \"0.65\"\} > tmp.metric'
dvc metrics show tmp.metric  # works well
dvc metrics modify tmp.metric --type json
dvc metrics show tmp.metric

Error: Unable to read metric in 'tmp.metric': 'NoneType' object is not subscriptable

Having any troubles? Hit us up at dvc.org/support, we are always happy to help!
Error: Failed to show metrics: File 'tmp.metric' does not exist

Having any troubles? Hit us up at dvc.org/support, we are always happy to help!
```

Also note double "Having any troubles" message.

**Comment by efiop** [comment]
The original bug was caused by incorrect command escaping, which resulted in trash being added to tmp.metric, causing jsonpath_rw module to error-out.

**Comment by efiop** [comment]
My bad, I forgot to run `-t json`. Can confirm this bug.

**Comment by mroutis** [comment]
I couldn't confirm this issue :thinking:

**Comment by efiop** [comment]
Fixed by #1709
